utils/fstack.c

```diff
@@ -503,7 +503,7 @@ int fstack_setup_filters(struct uftrace_opts *opts, struct uftrace_data *handle)
 
 /**
  * fstack_get - retrieve func_stack entry in a task
- * @task - tracee task
+ * @task - trace task
```

Comment:
Ah. `tracee` might be correct and used on purpose. @namhyung Could you please confirm this?

Reply:
Yeah, it was intended.

Reply:
I understand that `tracee` is the opposite of `tracer`.